docs/source/user_guide.rst

```diff
@@ -0,0 +1,184 @@
+User Guide
+==========
+
+Imaging inverse problems are described by the equation :math:`y = \noise{\forw{x}}` where
+:math:`x` is an unknown image we want to recover,
+:math:`y` are the observed measurements, :math:`A` is a
+deterministic (linear or non-linear) operator capturing the physics of the acquisition and
+:math:`N` characterizes the noise affecting the measurements.
+
```

Comment:
```suggestion
All the tools from the library, from measurement operator to restoration methods, are implemented as torch modules and therefore natively support backpropagation.
```

Reply:
should we move this to the training part?